### ISSUE TITLE
Questions about data set partitioning

### ISSUE THREAD

**Issue by pisces365** [turingmotors/NuScenes-MQA#2]
Is the QA extracted using the officially provided training set and test set partitioning method like this?

```python
from nuscenes.utils import splits
nusc = NuScenes(version='v1.0-trainval', dataroot='YOUR_DATA_PATH', verbose=True)
train_scenes = splits.train
valid_scenes = splits.val
```

Thank you for your great work!
